src/main/java/nl/tudelft/contextproject/model/level/util/RoomExitPoint.java

```diff
@@ -0,0 +1,23 @@
+package nl.tudelft.contextproject.model.level.util;
+
+/**
+ * Exit point of a room.
+ * 
+ * Used for determining if a door connection point has been used.
+ */
+public class RoomExitPoint extends DoorLocation {
```

Comment:
Or Interfaces, it does not matter what we use here, but inheritance makes a lot of stuff easy to check and test, also I'm against adding to many methods to enums, which should be in place and DoorLocations have to hold extra information in later stages of the algorithm